src/main/kotlin/com/refinedmods/refinedstorage/item/Wrench.kt

```diff
@@ -0,0 +1,40 @@
+package com.refinedmods.refinedstorage.item
+
+import com.refinedmods.refinedstorage.RS
+import com.refinedmods.refinedstorage.api.network.INetwork
+import com.refinedmods.refinedstorage.util.NetworkUtils.getNodeFromBlockEntity
+import com.refinedmods.refinedstorage.util.WorldUtils.sendNoPermissionMessage
+import com.thinkslynk.fabric.annotations.registry.RegisterItem
+import com.thinkslynk.fabric.generated.MyItemGroups
+import net.minecraft.block.BlockState
+import net.minecraft.item.Item
+import net.minecraft.item.ItemUsageContext
+import net.minecraft.util.ActionResult
+import net.minecraft.util.BlockRotation
+
+@RegisterItem(RS.ID, "wrench")
+class WrenchItem : Item(Settings().group(MyItemGroups.CURED_STORAGE).maxCount(1)) {
+
+    override fun useOnBlock(ctx: ItemUsageContext): ActionResult {
+        val player = ctx.player ?: return ActionResult.FAIL // i don't think this is possible in forge?
```

Comment:
Reasonable, but there is the occasional mod that allows you to place a tool in a block and that block uses it as though it were a player. So we might not want this to fail.

Reply:
But what about the securitymanager then?

Reply:
Not sure. We'll cross that bridge later, if it ever comes. This looks correct for now.